app/javascript/controllers/video_player_controller.js

```diff
@@ -61,4 +64,26 @@ export default class extends Controller {
       this.player.seekTo(time)
     }
   }
+
+  createOpenInYouTube () {
+    const videoContainer = this.playerTarget
+    const videoId = videoContainer.getAttribute('data-youtube-id')
+
+    const handleYouTubePlay = () => {
+      window.open(`https://www.youtube.com/watch?v=${videoId}`, '_blank')
+      this.player.pause()
+    }
+
+    const videoPlayerIcon = document.createElement('button')
+    videoPlayerIcon.className = 'v-openInYouTube v-controlButton'
+    videoPlayerIcon.innerHTML = youtubeSvg
+    videoPlayerIcon.addEventListener('click', handleYouTubePlay)
+
+    const externalPlayButton = document.querySelector('.v-playOnYoutubeButton')
+    if (externalPlayButton) {
+      externalPlayButton.addEventListener('click', handleYouTubePlay)
+    }
```

Comment:
Sorry, I have one more thing:

I was just wondering, is there a reason why this couldn't be a regular `<a>` tag? 

```suggestion
    const videoId = this.playerTarget.Container.getAttribute('data-youtube-id')
  
    const videoPlayerIcon = document.createElement('a')
    videoPlayerIcon.className = 'v-openInYouTube v-controlButton'
    videoPlayerIcon.innerHTML = youtubeSvg
    videoPlayerIcon.href = `https://www.youtube.com/watch?v=${videoId}`
    videoPlayerIcon.target = '_blank'
    videoPlayerIcon.dataset.action = "click->video-player#pause"
```

Similarly in the `app/views/talks/_talk.html.erb`, then it wouldn't need to have this dislocated behavior of attaching the onlick callback.

Reply:
when I was doing the implementation I checked #75 and via inspect element I found that the volume and full screen are implemented as a buttons so that's why I chose the current implementation.

Reply:
> Similarly in the `app/views/talks/_talk.html.erb`, then it wouldn't need to have this dislocated behavior of attaching the onlick callback.

not sure what you mean here

Reply:
I think for the other controls it would make sense for them to be button tags, but for a link opening a new website we should be using a tag that's meant for that. 

Doing it as a button also breaks the command+click behaviour for clicks.

So if it possible I would love to have this a a regular `<a>` tag

Reply:
Of course I like the implementation better as well. Better readability wise too.

However, one issue I'm noticing is that the icon is no longer consistent with the other icons based on screen size. Maybe we can wrap the button with <a> and put the icon inside? I think it's the best of both worlds.

Reply:
Sure, if that works that's fine too 👍🏼

Reply:
I cant really test `cmd+click` behavior on my device. Please let me know if you can test the behavior on your side, if it doesn't work I'll update the implementation.